itou/openid_connect/pro_connect/views.py

```diff
@@ -105,7 +105,7 @@ def _add_user_kind_error_message(request, existing_user, new_user_kind):
 def pro_connect_authorize(request):
     # Start a new session.
     user_kind = request.GET.get("user_kind")
-    previous_url = request.GET.get("previous_url", reverse("search:employers_home"))
+    previous_url = get_safe_url(request, "previous_url", fallback_url=reverse("search:employers_home"))
     next_url = request.GET.get("next_url")
     if next_url and not url_has_allowed_host_and_scheme(next_url, settings.ALLOWED_HOSTS, request.is_secure()):
```

Comment:
J'ai hésité à faire un:
```suggestion
    next_url = get_safe_url(request, "next_url")
    if request.GET.get("next_url") and not next_url:
```

Reply:
Je trouve que c’est moins clair.